### PR TITLE
[Fix] PM2.5 compensated show on display

### PR DESCRIPTION
**Pull request by pnt325**
Update missing add PM2.5 compensated before show on display. #221

### CODE REVIEW

File: src/AgOledDisplay.cpp [hunk 1]
```diff
@@ -307,20 +307,21 @@ void OledDisplay::showDashboard(const char *status) {
       int pm25 = value.pm25_1;
       if (config.hasSensorSHT) {
         pm25 = ag->pms5003.compensated(pm25, value.Humidity);
+        logInfo("PM2.5:" + String(value.pm25_1) + String("Compensated:") + String(pm25));
       }
       DISP()->setFont(u8g2_font_t0_22b_tf);
       if (config.isPmStandardInUSAQI()) {
-        if (utils::isValidPMS(value.pm25_1)) {
-          sprintf(strBuf, "%d", ag->pms5003.convertPm25ToUsAqi(value.pm25_1));
+        if (utils::isValidPMS(pm25)) {
```
Comment (nick-4711) [inline src/AgOledDisplay.cpp:314]: the method name isValidPMS is misleading, propose to rename to simply isValid()

File: src/AgOledDisplay.cpp [hunk 2]
```diff
@@ -307,20 +307,21 @@ void OledDisplay::showDashboard(const char *status) {
       int pm25 = value.pm25_1;
       if (config.hasSensorSHT) {
         pm25 = ag->pms5003.compensated(pm25, value.Humidity);
```
Comment (nick-4711) [inline src/AgOledDisplay.cpp:309]: the method name should be renamed to compensate()

File: src/AgOledDisplay.cpp (hunk 1 above)
Comment (pnt325) [reply src/AgOledDisplay.cpp:314]: it's the API on `utils` can not use `isValid` cause there are multiple API check sensor valid like `CO2`, `Temperature`, etc.

Comment (nick-4711) [reply src/AgOledDisplay.cpp:314]: OK, then I propose to write

isValidPm()

Comment (pnt325) [reply src/AgOledDisplay.cpp]: @nick-4711 
Updated 753f22923caa97cf0630c5a8da1ec6c6f44cb47e

**Pull request merged**